docs/msgs_package.md

```diff
@@ -22,7 +21,34 @@ For the encoder data stream the `edge_timestamp` is expected to be a constant ti
 
 <b>Note:</b>
 
-The LuciJoystickScaling message has the joystick values with the LUCI scaling applied. These are the values most recently used to control the chair and have already been sent to the wheelchairs drive controller. Please note this message also contains the scaling zone that was used for these outputs.
+The LuciJoystickScaling message has the joystick values with the LUCI scaling applied as well as the scaling applied. These are the values most recently used to control the chair and have already been sent to the wheelchairs drive controller. Please note this message also contains the scaling zone that was used for these outputs.
+
+Input source parameter is added to LuciJoystick and LuciJoystickScaling. This parameter denotes which input source has been set by the user. If this is used in any place to send commands to LUCI, make sure it is set to the same input as the desired/set state on LUCI or LUCI will disregard the command.
+
+The input source are mapped as following:
+
+| Integer | Input Source      |
+| ------- | ----------------- |
+|    0    |  RampAssist       |
+|    1    |  AutonomousRemote |
+|    2    |  WDI              |
+|    3    |  ChairVirtual     |
```

Comment:
Could we just call `ChairVirtual` and `ChairPhysical` like `Chair1` and `Chair2` or something? I'd like to avoid there being confusion around the difference between those since it really shouldn't matter here.

Reply:
I suppose it could. I'm fine with leaving it and potentially updating docs if there is confusion in the future.